src/source/SwiftBridgingHeaderGenerator.scala

```diff
@@ -38,8 +38,23 @@ class SwiftBridgingHeaderGenerator(spec: Spec) extends Generator(spec) {
 }
 
 object SwiftBridgingHeaderGenerator {
-  def writeAutogenerationWarning(writer: Writer) {
+  
+  val bridgingVarPrefix = (s: String) => s.split('-').mkString("_")
+
+  def writeAutogenerationWarning(name: String, writer: Writer) {
+    val varPrefix = bridgingVarPrefix(name)
     writer.write("// AUTOGENERATED FILE - DO NOT MODIFY!\n")
     writer.write("// This file generated by Djinni\n\n")
+    writer.write("// " + varPrefix + ".h\n")
+    writer.write("// " + varPrefix + "\n\n")
+  }
+
+  def writeBridgingVars(name: String, writer: Writer) {
+    val varPrefix = bridgingVarPrefix(name)
+    writer.write("#import <Foundation/Foundation.h>\n\n")
+    writer.write("//! Project version number for " + varPrefix +".\n")
+    writer.write("FOUNDATION_EXPORT double " + varPrefix + "VersionNumber;\n\n")
+    writer.write("//! Project version string for" + varPrefix +".\n")
```

Comment:
I don't like the way this variable is named. In this case: `"..." + varPrefix +"..."`, it's not used as a prefix.

Although it happens to be a prefix in the case of `VersionNumber`, I'd still prefer to remove the `prefix` in the var name, to make it something like `bridgingHeaderVarName`.